ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -149,7 +149,7 @@ tests:
         extract should extract\| Image info should display\| Image layer subresource\|
         oc tag should change image\| when installed on the cluster should\| OpenShift
         alerting rules\| The HAProxy router should\| egressrouter cni resources\|
-        pod should start\| pod sysctls
+        pod should start\| pod sysctls\| events should not repeat pathologically
```

Comment:
IDK if this PR is just to trigger rehearsals, which is fine, but I would say that we probably don't want to skip these tests since thrown events may indicate issues with any of the other underlying tests. :)

Reply:
This is my first PR against this repo. I just test it out and try to understand the development process and 
setup.